labelbox/schema/project.py

```diff
@@ -312,18 +316,66 @@ def setup(self, labeling_frontend, labeling_frontend_options):
         timestamp = datetime.now(timezone.utc).strftime("%Y-%m-%dT%H:%M:%SZ")
         self.update(setup_complete=timestamp)
 
+    def validate_labeling_parameter_overrides(self, data):
```

Comment:
Its not worth changing it. Especially since the pydantic error messages will be a little less specific. We should probably use pydantic for defining the types to improve input validation further in the future though.